src/ontology/uberon-edit.obo

```diff
@@ -128248,20 +128248,19 @@ relationship: part_of UBERON:0002299 ! alveolus of lung
 [Term]
 id: UBERON:0008874
 name: pulmonary acinus
-def: "The part of the airway consisting of a respiratory bronchiole and all of its branches." [MP:0010911, Wikipedia:Acinus#The_Lungs]
+def: "The largest lung unit in which all airways participate in gas exchange, and it is composed of respiratory bronchioles, alveolar ducts, and alveoli." [MP:0010911, PMID:2014492, Wikipedia:Acinus#The_Lungs]
```

Comment:
The original definition seems more specific-- it uses the words "part of the airway" instead of the vague "lung unit". The paper acknowledges that this (the added text) is a functional definition and the aim of the study seems an attempt to remove the ambiguity from this definition.

Maybe @JoshuaFortriede can offer more insight.

Reply:
I am reaching out to a lung expert on this. From my current understanding of the pulmonary acinus, I like the new definition. This structure/unit is not part of the airway, it contains a specific section of the airway. Additionally, the pulmonary acinus contains more than just the airway, there is supporting tissue, vasculature, etc. As such, I do believe the original definition is not only lacking but technically inaccurate. I will update when I know more.

Reply:
Thank you Joshua, I will wait for the update before merging.